component/core/src/main/java/org/exoplatform/social/core/jpa/search/ProfileSearchConnector.java

```diff
@@ -142,10 +144,13 @@ private int getCount(String jsonResponse) {
       String avatarUrl = (String) hitSource.get("avatarUrl");
       String email = (String) hitSource.get("email");
       String identityId = (String) ((JSONObject) jsonHit).get("_id");
-      String aboutMe = (String) hitSource.get("aboutMe");
       identity = new Identity(OrganizationIdentityProvider.NAME, userName);
       identity.setId(identityId);
       p = new Profile(identity);
+      Profile profile = getIdentityProfile(userName);
+      if (profile == null) {
```

Comment:
@hakermi This should not b e a normal case, do you have a steps to reproduce such a behaviour ?